x-pack/plugins/observability_solution/observability_shared/public/hooks/use_editable_settings.tsx

```diff
@@ -88,8 +92,16 @@ export function useEditableSettings(settingsKeys: string[]) {
         await Promise.all(arr);
         setForceReloadSettings((state) => ++state);
         cleanUnsavedChanges();
+        if (updateErrorOccurred) {
+          throw new Error('One or more settings updates failed');
```

Comment:
How is this error handled? Do we show it in the UI?

Reply:
It depends on implementation.  For AI Assistant and APM the [errors are handled and display a toast](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/settings_tab/ui_settings.tsx#L39-L47).  In Infra, they aren't being handled and the behaviour is unchanged: no error is shown and a success toast is shown.  I'm going to open an issue for them.